vllm/entrypoints/openai/api_server.py

```diff
@@ -894,6 +902,9 @@ async def init_app_state(
     ) if model_config.runner_type == "transcription" else None
     state.task = model_config.task
 
+    state.server_load_metrics_lock = threading.Lock()
```

Comment:
We do not need locks. This is not threaded code this is asyncio code so only one coroutine can run at a time